en/guide/overview.md

```diff
@@ -16,7 +16,7 @@ CSD supports the following key features:
 * Automatically attaches [compatible cameras](#supported_cameras) connected to the Linux computer when it is started.
 * RTSP video streaming from *all* connected cameras (for consumption by GCS or other video players).
 * RTSP video stream advertising/discovery using Avahi.
-* [MAVLink Camera Protocol](#mavlink_support) support for up to 5 cameras, enabling image/video capture and storage, and querying/setting camera options.
+* [MAVLink Camera Protocol](#mavlink_support) support for up to 6 cameras, enabling image/video capture and storage, and querying/setting camera options.

```

Comment:
Should image capture, video capture be part of key feature of CM?

Reply:
Yes. They are, but they are part of "MAVLink support". That is why the doc says "... support for up to 6 cameras, enabling image/video capture and storage, and querying/setting camera options."